bbc/simorgh
# WSTEAM1-971 - Add Uploader translations

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-971 

Overall changes
======
- Adds translations for services that had translations in the old Uploader
- Adds fallback translations to English when full or partial translations are missing
- Wires up new Uploader components to use these translations
- Missing translations are being collated and will be brought in in a separate PR

Code changes
======
- Adds new `ugc` translation and type
- Adds `ServiceContext` to relevant Uploader form components and screens to use translations

Testing
======
1. Visit the form screens in Storybook, for example the 'Uploading Screen': https://wsteam1-971-add-uploader-translations--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-ugc-page--uploading-screen
2. Change the service using the global service selector at the the top of the page (the global icon)
3. Change the service to one of the languages that has translations in this PR, for example `arabic`
4. Confirm that the translations are used on the page where there is a translation, falling back to English where there is not
5. Visit the form screen and confirm validation is translated https://wsteam1-971-add-uploader-translations--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-ugc-page--form-with-file-upload&globals=service.service:arabic 

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
